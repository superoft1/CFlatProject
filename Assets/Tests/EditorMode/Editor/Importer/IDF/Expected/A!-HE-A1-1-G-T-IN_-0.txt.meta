fileFormatVersion: 2
guid: 888ea69f8b801ca4ca06012f975dcb2a
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
