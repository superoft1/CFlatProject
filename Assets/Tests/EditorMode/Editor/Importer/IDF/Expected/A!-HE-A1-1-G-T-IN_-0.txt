WeldNeckFlange, , (1.0, 0.0, 0.0), (-373.7, 14.5, 0.9), False
(-373.6, 14.5, 0.9)
(-373.7, 14.5, 0.9)
FreeVertex: 1
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-373.5, 14.5, 0.9), False
(-373.6, 14.5, 0.9)
(-373.4, 14.5, 0.9)
FreeVertex: 0
Pipe, , (1.0, 0.0, 0.0), (-373.4, 14.5, 0.9), False
(-373.4, 14.5, 0.9)
(-373.3, 14.5, 0.9)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-373.3, 14.5, 0.9), False
(-373.3, 14.5, 0.9)
(-373.3, 14.5, 0.9)
(-373.3, 14.5, 0.6)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-373.3, 14.5, 0.5), False
(-373.3, 14.5, 0.6)
(-373.3, 14.5, 0.5)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-373.3, 14.5, 0.5), False
(-373.3, 14.5, 0.4)
(-373.3, 14.5, 0.5)
FreeVertex: 1
Pipe, , (1.0, 0.0, 0.0), (-373.2, 14.5, 0.9), False
(-373.3, 14.5, 0.9)
(-373.1, 14.5, 0.9)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-372.2, 14.5, 0.9), False
(-373.1, 14.5, 0.9)
(-372.2, 14.5, 1.8)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-372.2, 14.5, 2.0), False
(-372.2, 14.5, 1.8)
(-372.2, 14.5, 2.2)
FreeVertex: 0
WeldNeckFlange, , (0.0, 0.0, 1.0), (-372.2, 14.5, 2.3), False
(-372.2, 14.5, 2.4)
(-372.2, 14.5, 2.2)
FreeVertex: 1
FreeVertexTotal: 3
